src/Stratis.Bitcoin.Tests/Builder/FullNodeBuilderTest.cs

```diff
@@ -122,7 +122,7 @@ public void BuildWithInitialServicesSetupConfiguresFullNodeUsingConfiguration()
             this.fullNodeBuilder.ConfigureServiceProvider(e =>
             {
                 var settings = e.GetService<NodeSettings>();
-                settings.Testnet = true;
+                settings.Agent = "abc";
```

Comment:
Why is this needed?

Reply:
Going by the legacy code one of the purposes of the test case seems to be to prove that the node settings can be changed. Since we no longer have a Testnet variable I exchanged that for a change to the Agent variable.

Reply:
yeah please, and in BuildConfiguresFullNodeUsingConfiguration as well.

Reply:
Done.